app_server/app/adapters/socket/RoomsManager.js

```diff
@@ -106,6 +118,10 @@ Class.removeRoom = function(roomName) {
   delete this.rooms[roomName];
 };
 
+Class.getRooms = function() {
+  return this.rooms;
+};
+
```

Comment:
Will we be using getters and setters for most things?
Accessing it directly doesn't seem like much of an issue.